### ISSUE TITLE
Add FourCastNet to the default Benchmarks

### ISSUE THREAD

**Issue by itsgifnotjiff** [google-research/weatherbench2#44]
It seems that FourcastNet, Pangu and GraphCast are the leaders in Climate ML and so it would be interesting to add FourCastNet to the benchmark if possible. Thank you for your work.

**Comment by raspstephan** [comment]
Thanks for the comment. It will happen but give us a few weeks :)

**Comment by itsgifnotjiff** [comment]
Thank you very much Mr. @raspstephan . Your work has a big fan at Environment and Climate Change Canada and hopefully a contributor in the near future.

**Comment by raspstephan** [comment]
Awesome, let me know if you need any help or have any questions.

**Comment by raspstephan** [comment]
Yes, we don't have the bandwidth to run the open-source models ourselves but we reached out to the FourCastNet team and they mentioned that they would provide us with forecasts once the new version is out.
